src/components/NumericInput/index.js

```diff
@@ -77,6 +77,7 @@ class NumericInput extends TextInput {
                 this._sliderControl.dom.requestPointerLock();
                 this._sliderMovement = 0.0;
                 this._sliderPrevValue = this.value;
+                this._sliderUsed = true;
```

Comment:
This looks good but could we make the `_sliderUsed` variable a local boolean instead of a NumericInput property as it's not used anywhere else?

Reply:
Good point. I will update it.